tests/gold_tests/bigobj/bigobj.test.py

```diff
@@ -52,65 +47,88 @@
 )
 
 ts.Disk.remap_config.AddLine(
-    'map https://localhost http://localhost'
+    f'map https://localhost:{ts.Variables.ssl_port} http://localhost:{ts.Variables.port}'
+)
+ts.Disk.remap_config.AddLine(
+    f'map https://localhost:{ts.Variables.ssl_portv6} http://localhost:{ts.Variables.port}'
 )
-
-# Set up to check the output after the tests have run.
-#
-log_id = Test.Disk.File("log2.txt")
-log_id.Content = "log2.gold"
 
 # Size of object to get.  (NOTE:  If you increase this significantly you may also have to increase cache
 # capacity in tests/gold_tests/autest-size/min_cfg/storage.config.  Also, for very large objects, if
 # proxy.config.diags.debug.enabled is 1, the PUSH request will timeout and fail.)
 #
 obj_kilobytes = 10 * 1024
+obj_bytes = obj_kilobytes * 10
+header = "HTTP/1.1 200 OK\r\nContent-length: {}\r\n\r\n".format(obj_bytes)
+
+
+def create_pushfile():
+    f = open(Test.RunDirectory + "/objfile", "w")
+    f.write(header)
+    f.write("x" * obj_bytes)
+    f.close()
+    return True
+
 
 tr = Test.AddTestRun("PUSH an object to the cache")
 # Delay on readiness of TS IPv4 ssl port
-tr.Processes.Default.StartBefore(ts)
-#
+tr.Processes.Default.StartBefore(ts, ready=lambda: create_pushfile())
 # Put object with URL http://localhost/bigobj in cache using PUSH request.
-tr.Processes.Default.Command = (
-    f'./push_request {obj_kilobytes} | nc localhost {ts.Variables.port}'
```

Comment:
This runs netcat with no arguments.  But there's still some incompatibility with Ubuntu retro netcat?

Reply:
Since curl does support PUSH, that seems more reliable and maintainable than relying on netcat. 

The specific failure I was seeing on u22 was with the last testcase
> file /home/ubuntu/ats10/build-mydev/tests/_sandbox/bigobj/_output/6-tr-Default/stream.stdout.txt : The PUSH request should have received a 403 response. - ^[[0m^[[31mFailed^[[0m^[[1m
          Reason: Contents of /home/ubuntu/ats10/build-mydev/tests/_sandbox/bigobj/_output/6-tr-Default/stream.stdout.txt did not contains expression: "403 Access Denied"

Specifically the stream.stdout.txt file was empty.  I assume there is a connection close difference or something between the distros.  Rather than debugging that, since nc isn't really needed, I felt it would be easier to deal with by using curl instead.

Reply:
I only get this far:
```
   Run: GET bigobj: TLS, HTTP/2, IPv6: Failed
     Starting TestRun 4-tr : No Issues found - Passed
        Reason: Started!
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
```
The failure is here:  https://github.com/apache/trafficserver/blob/870e242eee6d382750ad383abde03e416314bba0/tests/gold_tests/bigobj/check_ramp.c#L41

I'll approve this, but I feel like we're whistling past the graveyard a bit.  Ideally, our white box regression testing should be robust on all supported platforms.